index.html

```diff
@@ -27,17 +27,42 @@
       </p>
     </section>
     <section id="sotd">
-        <p>This is a draft for a future SocialCG report.</p>
+        <p>This is a draft of a future SocialCG report.</p>
       </section>
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-        Data portability is the ability of a user to move their data on one
+        Data portability is the ability of a user to move their data from one
         system to another system. This includes moving between ActivityPub
         implementations, moving to different installations of the same
         software, or changing domains.
       </p>
+
+    <section class="informative">
+      <h2>Terminology</h2>
+
+      <p>
+        The following terms are used to describe concepts in this report.
+      </p>
+
+      <dl class="termlist">
+        <dt><dfn data-lt="instance|instances">instance</dfn></dt>
+        <dd>An informal name for an ActivityPub <a>server</a>, specifically,
+          a particular instance of a given ActivityPub server implementation.
+          In current implementations, instances are typically bound to a given
```

Comment:
what does "bound to" mean here, effectively? that a domain controlled by the controllers of the instance is used as the domain of all `id`s and other URIs? something more esoteric about reputation and federation/allowlisting? both?

Reply:
>  that a domain controlled by the controllers of the instance is used as the domain elemtn of all the URLs that serve as ids and other URIs?

Yes, that! AND allow-listing. We can define the binding later on.